File: app/lib/models/announcement.dart
class Announcement {
  final String content;
  final String contentHtml;

  Announcement({
    required this.content,
    required this.contentHtml,
  });

  factory Announcement.fromJson(Map<String, dynamic> json) {
    return Announcement(
      content: json['content'] ?? '',
      contentHtml: json['content_html'] ?? json['contentHtml'] ?? '',
    );
  }
}

File: backend/models/announcement.go
package models

import (
	"time"
)

type Announcement struct {
	ID          uint      `gorm:"primarykey" json:"id"`
	Content     string    `gorm:"type:text" json:"content"`
	ContentHTML string    `gorm:"type:text" json:"content_html"`
	UpdatedAt   time.Time `json:"updated_at"`
}
